layers/+spacemacs/spacemacs-navigation/packages.el

```diff
@@ -323,6 +323,7 @@
     :defer t
     :init
     (progn
+      (spacemacs/set-leader-keys "hj" 'info-display-manual)
```

Comment:
It seems like `helm-info-woman` is a better choice if `helm` is being used.

Ivy supports Info nodes out-of-box but I'm not sure whether there already exists commands similar to those of helm's. In any case we can fall back to `info-display-manual`.

---

Also, I'm thinking about another possibility for key bindings:

|Command                |Current Keybinding|Proposed Keybinding|
|-----------------------|------------------|-------------------|
|helm-man-woman         |`SPC h m`         |`SPC h m`          |
|helm-switch-major-mode |`SPC h M`         |`SPC f m`          |
|helm-info-woman        |                  |`SPC h M`          |

`SPC f m` for switching the major mode seems more natural then `SPC h M`.
Alternatively, `SPC b M` is also a valid choice and probably better if we expect
more file-related keybindings to be added. (Say `SPC f m` for a hypothetical `file-modify`).

Reply:
Sorry, I have placed the binding on `SPC h j`. I was just commenting that it might also as well be bound under `SPC h M` (because the `helm-major-mode` thing is rather redundant)...

Also, the `info-display-manual` jumps to `info` files not to `man` files...

Reply:
> Also, the info-display-manual jumps to info files not to man files...

I know, that what exactly helm-info-woman does.

Reply:
Sorry, I mean texinfo pages or whatever they are called (e.g. the emacs, calc, magit manual etc.)

I find it annoying to always have to navigate there via the `C-h i`

Reply:
`helm-info-woman` jumps to info nodes, not man pages.

Reply:
Yes I see, but it is annoying to jump to nodes, because I do not know how their namer. Moreover, it seems that `helm-info-woman` includes only very few nodes (e.g. try to jump to `transient` `classes and methods`, or any `calc` node).
I would prefer to just jump to the manual (e.g. elisp, calc, magic), but `helm-info-woman` does not offer that functionality. Also, I put this binding under `SPC h j`, while you are mentioning only bindings ending with `m/M`. So, your suggestions and this PR are not interfering.

Reply:
I am using Ivy b.t.w., but otherwise it sounds reasonable to use `woman`' instead of `man`, and to add the `info` binding. Once more, I find the `helm-switch-major-mode` binding quite redundant, and otherwise probably more logical under `SPC b M`.

Reply:
agreed. just make a new PR to move it to SPC b